.github/workflows/ipfs-deploy.yml

```diff
@@ -65,3 +65,71 @@ jobs:
       - name: Publish to nft.storage
         run: |
           docker run -e NFTSTORAGE_API_KEY=${{ secrets.NFTSTORAGE_API_KEY }} $IMAGE_RELEASE_ID nft.storage
+
+      - name: Create a release
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          REF_NAME: ${{ github.ref_name }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
```

Comment:
These are both available to all workflow steps automatically.  No need to set them in an `env` section.  Secrets are special and must be specified in the `env` section to be available in steps.
```suggestion
```
Note: You'll need to rename to `GITHUB_REF_NAME` elsewhere.

Reply:
applied on https://github.com/DarkFlorist/lunaria/pull/261/commits/72fe8eac662858e8ab713b42b4c7907c7263273f